src/components/analyser/search/SearchBar.tsx

```diff
@@ -56,35 +60,46 @@ const SearchBar: React.FC = () => {
     fetch();
   }, []);
 
-  const [open, setOpen] = useState(false);
   return (
     <div style={{ width: 300 }}>
       {stocks && (
         <Autocomplete
           id="search"
           freeSolo
+          inputValue={inputValue}
+          disableClearable
           open={open}
           options={stocks}
           getOptionLabel={(option: API.Stock) =>
-            `${option.symbol} ${option.name}${option.isin}${option.wkn}`
+            `${option.symbol}: ${option.name}`
```

Comment:
There's a problem now that stocks cannot be searched by ISIN or WKN.

Reply:
Should be fixed now